pkg/bootstrap/docker/openshift/pvsetup.go

```diff
@@ -0,0 +1,125 @@
+package openshift
+
+import (
+	"fmt"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kbatch "k8s.io/kubernetes/pkg/apis/batch"
+	kclientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+
+	"github.com/openshift/origin/pkg/bootstrap/docker/errors"
+	"github.com/openshift/origin/pkg/client"
+)
+
+const pvCount = 1000
+const createPVScript = `#/bin/bash
+
+set -e
+
+function create_pv() {
+  BASEDIR=${1}
+  NAME=${2}
+cat <<EOF
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: ${NAME}
+  labels:
+    volume: ${NAME}
+spec:
+  capacity:
+    storage: 100Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: ${BASEDIR}/${NAME}
+  persistentVolumeReclaimPolicy: Retain
```

Comment:
@bparees I tested this and it does work. Once you delete a PVC, the contents of the pv directory are wiped out. Trying to understand if this is what we should be using ('Recycle' instead of 'Retain')